grpc-interface-support/test/integration/test_integration_python.py

```diff
@@ -73,7 +76,30 @@ async def wait_until_ready(self):
         async def stop(self):
             pass
 
-    middleware = TestMiddleware()
+    print("============= BUILDING SERVER ===================")
+    os.chdir(os.environ["SERVICE_SERVER_ROOT"])
+    assert subprocess.check_call(["velocitas", "init", "-v"]) == 0
+
+    from seats_service_sdk.seats_pb2_grpc import SeatsServicer
+    from seats_service_sdk.SeatsServiceServerFactory import SeatsServiceServerFactory
+
+    middleware = TestMiddleware(TestServerServiceLocator())
```

Comment:
Use native middleware instead, it is doing the same thing.

Reply:
How will this work with the tests? The ServiceLocator can't be injected with a mocked one where I override the `get_service_location`.

```
class NativeMiddleware(Middleware):
    """Native middleware implementation."""

    def __init__(self) -> None:
        super().__init__()

        self.type = MiddlewareType.NATIVE
        self.service_locator = NativeServiceLocator()

        _address = self.service_locator.get_service_location("mqtt")
        _port = urlparse(_address).port
        _hostname = urlparse(_address).hostname
        self.pubsub_client = MqttClient(_port, _hostname)
```

Reply:
I agree with @Chrylo